### PR TITLE
removed unused pngs from drawable

### PR DESCRIPTION
**Pull request by shubhamdamani**
removed unused pngs from drawable.

### CODE REVIEW

**Comment by shubhamdamani** [comment]
how to create multiple pull requests for different issues @prabhakar267. I have worked on two issues but they got committed in one.

**Comment by Swati4star** [comment]
@shubhamdamani Please resolve the conflicts.

**Comment by prabhakar267** [comment]
@shubhamdamani Please fix your build https://travis-ci.org/project-travel-mate/Travel-Mate/builds/593144428?utm_source=github_status&utm_medium=notification

File: Android/build.gradle [hunk 1]
```diff
@@ -9,15 +9,18 @@ buildscript {
         google()
     }
     dependencies {
+
```
Comment (Swati4star) [inline Android/build.gradle]: why these changes? please revert this file
